framework/sessions/__init__.py

```diff
@@ -95,7 +95,7 @@ def set_session(session):
     sessions[request._get_current_object()] = session
 
 
-def create_session(response, data=None):
+def create_session(response, data=None, user_agent=None):
```

Comment:
It doesn't look like `user_agent` is used in this function, or ever passed in when it's called